packages/lisk-p2p/test/integration/p2p.ts

```diff
@@ -1,40 +1,40 @@
-import { P2P } from '../../src/index';
+// import { P2P } from '../../src/index';
 
-describe('Integration tests for P2P library', () => {
-	const NETWORK_PEER_COUNT = 10;
-	let blockchainP2PList: ReadonlyArray<P2P> = [];
+// describe('Integration tests for P2P library', () => {
```

Comment:
instead of commenting out, u can use `skip`

Reply:
yeah I forgot that I can use `skip`, also, I will take it out from here and do it in a separate PR